apps/server/src/migrations/mikro-orm/Migration20240604131554.ts

```diff
@@ -0,0 +1,44 @@
+import { Migration } from '@mikro-orm/migrations-mongodb';
+import * as process from 'node:process';
+
+export class Migration20240604131554 extends Migration {
+	async up(): Promise<void> {
+		// eslint-disable-next-line no-process-env
+		if (process.env.SC_THEME === 'n21') {
+			await this.driver.nativeInsert('instance-configs', {
+				name: 'nbc',
+			});
+			console.info('Instance config was added for nbc');
+		}
+
+		// eslint-disable-next-line no-process-env
+		if (process.env.SC_THEME === 'brb') {
+			await this.driver.nativeInsert('instance-configs', {
+				name: 'brb',
+			});
+			console.info('Instance config was added for brb');
+		}
+
+		// eslint-disable-next-line no-process-env
+		if (process.env.SC_THEME === 'thr') {
+			await this.driver.nativeInsert('instance-configs', {
+				name: 'thr',
+			});
+			console.info('Instance config was added for thr');
+		}
+
+		// eslint-disable-next-line no-process-env
+		if (process.env.SC_THEME === 'default') {
+			await this.driver.nativeInsert('instance-configs', {
+				name: 'default',
+			});
+			console.info('Instance config was added for default');
+		}
```

Comment:
Okay one other thing whar me wonder is at backup/setup/instance-configs.json there you add all 4 short's for the instance to the instance-configs and at this point only the one who is needed.
So is there an issue if there is more then one instance value at this collection or could it be als in this migration simple all 4?

Reply:
@mamutmk5 There should only be one entry on Prod or Ref, because an instance should not know anything about other instances. In backup/setup/instance-configs.json this is actually also the case, but maybe for dev this is ok, because a developer wants to test another instance locally????

Reply:
mh okay, works it with all in it or only one?

Reply:
Ok, we only need a collection and an ObjectId in the ticket. If we have concerns about the name, then I suggest throwing out the name. What do you think? @mamutmk5 @CeEv

Reply:
Or we take the name like it is... It is nothing what cant be changed anymore

Reply:
yes it's fine that we take it as it is